client/packages/common/src/ui/forms/Detail/InputWithLabelRow.tsx

```diff
@@ -51,8 +51,10 @@ export const DetailInputWithLabelRow: FC<InputWithLabelRowProps> = ({
   );
 };
 
-// Adds a final ":" to the label, but not if it already ends in "?" or ":"
+// Adds a final ":" to the label, but not if it already ends in "?" or ":", or
+// if there is no label value
 export const labelWithPunctuation = (labelString: string) => {
   if (/[\?:]$/.test(labelString)) return labelString;
+  if (labelString === '') return '';
```

Comment:
Didn't actually end up needing this, but it makes sense to leave -- we don't want to display a solitary ":" if the label text is empty.